v3/doio/kb38/kb38-01.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "DOIO KB38-01",
+  "vendorId": "0xD010",
+  "productId": "0x3801",

```

Comment:
I just picked up a KB38 and it seems like the vendorId it reports is `0xfeed` and the productId `0x6060`, which doesn't match this or any entry in this repo, so isn't picked up by `usevia.app`. The [json linked on the keebmonkey wiki](https://wiki.keebmonkey.com/en/products/mmp) _does_ have those vendor/product ids, and seems to work. Is there an newer/older version that also should get added here?

Reply:
@weswigham thanks for the heads up, I'll investigate and take action to either remove the JSON or block the fetching. it might be that the VID/PID check wasn't right. Unfortunately the PID/VID need to match the one in QMK so if that's not the case/they don't satisfy the VIA PR Checklist requirements the file will be removed

Reply:
It's [`0xFEED` in QMK](https://github.com/qmk/qmk_firmware/blob/master/keyboards/doio/kb38/info.json#L26) (with a differing productId, too)

Reply:
It was [changed on the develop branch](https://github.com/qmk/qmk_firmware/blob/develop/keyboards/doio/kb38/keyboard.json#L27) to match what was here, but it's still not what the device actually reports, so... 🤷‍♂️

Reply:
yeah, saw that. As per PR check requirements the PID/VID must match the one in QMK main so I went on and removed the board. Also, the general mismatch between what is merged and what was shipped form whoever manufactured the board doesn't help. They'll have to figure stuff out